msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -135,9 +135,13 @@ If your app makes use of [Content Security Policy](https://developer.mozilla.org
     |Teams web |5e3ce6c0-2b1f-4285-8d4b-75ee78787346 |
     |Office web  |4765445b-32c6-49b0-83e6-1d93765276ca|
     |Office desktop  | 0ec893e0-5785-4de6-99da-4ed124e5296c |
+    |Office mobile  | d3590ed6-52b3-4102-aeff-aad2292ab01c |
     |Outlook desktop, mobile | d3590ed6-52b3-4102-aeff-aad2292ab01c |
     |Outlook web | bc59ab01-8403-45c6-8796-ac3ef710b3e3|
 
+    > [!NOTE]
+    > Please note that some Microsoft apps share client IDs as this simplifies SSO and also enables a feature to more easily show up across different Microsoft apps
+
 ## Sideload your app in Teams
```

Comment:
[Line 165] After it's sideloaded to Teams, your personal tab is available in Outlook and Office. Be sure to sign in with the same credentials you used to sign in to Teams to sideload your app.

. . . could we add this:

After it's sideloaded to Teams, your personal tab is available in Outlook and Office. Be sure to sign in with the same credentials you used to sideload your app in Teams. When running the Office app for Android, you'll need to restart to use your personal tab app from the Office app.